### ISSUE TITLE
Shift + Tab not working on Windows for alacritty 0.6

### ISSUE THREAD

**Issue by ghillb** [alacritty/alacritty#4496]
Hello, for some reason `Shift + Tab` is not working in alacritty (it is working in powershell and other terminal windows on the same system).

Expected behavior: pressing shift + tab sends ^[[Z sequence to terminal.
Actual behavior: pressing shift + tab does not send anything to the terminal (`showkey -a` output is empty)

I tried this workaround: remap shift + tab in `alacritty.yml` and send sequence this way.
Sadly, it does not work and each of the following mappings sends `^[[` , which gets interpreted, and then an individual `Z`. 

```
  - { key: Tab, mods: Shift, chars: "^[[Z" }
  - { key: Tab, mods: Shift, chars: "\x1b\x5b\x5a" }
  - { key: Tab, mods: Shift, chars: "\x5e\x5b\x5b\x5a" }
```
### System

OS: Windows 10 x64 Build17763.1577
Version: alacritty 0.6.0 (04cbf76)

**Comment by chrisduerr** [comment]
> Shift + Tab not working on Windows for allacritty 0.6

So does it work with 0.5.0?

**Comment by ghillb** [comment]
@chrisduerr no it's the same for 0.5

**Comment by chrisduerr** [comment]
I'm pretty certain this is a ConPTY issue, so not up to us. Are you able to test any other terminal emulator on Windows that makes use of the ConPTY?

**Comment by ghillb** [comment]
You are probably right. I tried a terminal emulator called "Fluent Terminal" which uses ConPTY also, and the result was the same. 

Shift+Tab doesn't seem too obscure of a key combo, and I use it in many workflows. It's really a pity that alacritty on Windows has this limitation (although inherited from ConPTY).

I tried using `winpty_backend: true`, but it seems deprecated and is not working in 0.6 (alacritty hangs on start). In 0.5 it works but exhibits some other limitations that make it kinda awkward to use and being stuck at 0.5 doesn't seem to be a solution either..
